app/assets/stylesheets/users.scss

```diff
@@ -46,3 +49,15 @@ $red: #ff0000;
     margin: 0 0 30px;
   }
 }
+
+.col.l3.m5.s5.user-show {
+  height: 30em;
+  background-color: #a3cded;
+}
+
+.col.l9.m7.s7.user-show {
+  height: 30em;
```

Comment:
Properties should be ordered background-color, border-bottom, height, padding-bottom